infer/src/base/CommandLineOption.mli

```diff
@@ -126,7 +127,7 @@ val mk_symbol_opt :
 val mk_symbol_seq :
   ?default:'a list -> symbols:(string * 'a) list -> eq:('a -> 'a -> bool) -> 'a list ref t
 (** [mk_symbol_seq long symbols] defines a command line flag [--long <symbol sequence>] where
-    [<symbol sequence>] is a comma-separated sequence of [<symbol>]s such that [(<symbol>,_)] is an
+    [<symbol sequence>] is a comma-separated sequence of [<symbol>]-s such that [(<symbol>,_)] is an
```

Comment:
This hyphen is odd, what is the problem without it?

Reply:
If there is no `-` or `'` a space is added between the code block and the following text, but here we can just remove the `s` and the sentence still makes sense.